OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseTabsDashboardFragment.java

```diff
@@ -95,30 +97,36 @@ public static CourseTabsDashboardFragment newInstance(Bundle bundle) {
     public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
         courseData = (EnrolledCoursesResponse) requireArguments().getSerializable(Router.EXTRA_COURSE_DATA);
+        binding = FragmentCourseTabsDashboardBinding.inflate(inflater, container, false);
```

Comment:
better to `inflate` where it's required. cuz it leads to ambiguity.

Reply:
we will be using this view in whole class once the designs implementation of app_nav is completed.